packages/theme/src/types.ts

```diff
@@ -2,11 +2,5 @@ export type CSSColor =
     | `#${string}`
     | `rgb(${number}, ${number}, ${number})`
     | `rgba(${number}, ${number}, ${number}, ${number})`
-    | 'transparent';
-
-export const isCSSColor = (color: string): color is CSSColor => {
-    const cssColorRegex =
-        /^#([0-9a-fA-F]{3,8})$|^rgba?\((\d{1,3}),\s*(\d{1,3}),\s*(\d{1,3}),?\s*(\d{1,3})\)$/;
-
-    return cssColorRegex.test(color);
-};
+    | 'transparent'
+    | 'currentColor';
```

Comment:
Please update https://github.com/trezor/trezor-suite/blob/622098db3ff378fadbfd4871f6f919de810b99c1/suite-common/icons-deprecated/src/components/Icon.tsx#L21

Reply:
Would be nice to narrow `CSSColor` down to the colors from `palette`. But I assume something is blocking it?